desktop/hot-path.js

```diff
@@ -0,0 +1,6 @@
+import resolveAssets from './resolve-assets'
+const hot = process.env.HOT === 'true'
+
+export default path => {
+  return (hot ? 'http://localhost:4000/dist/' : resolveAssets('dist') + '/') + path
```

Comment:
We likely shouldn't, it'd be really hard to understand what version of what is running and it'd be slow